android/app/src/main/assets/custom/balloon.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
was there an ios issue being fixed here? Or why are the SVGs added here as well?